cscore/src/main/native/include/cscore_cpp.h

```diff
@@ -317,9 +317,15 @@ void SetSourceEnumPropertyChoices(CS_Source source, CS_Property property,
 CS_Sink CreateMjpegServer(std::string_view name, std::string_view listenAddress,
                           int port, CS_Status* status);
 CS_Sink CreateCvSink(std::string_view name, CS_Status* status);
+CS_Sink CreateCvSink(std::string_view name, VideoMode::PixelFormat pixelFormat,
+                     CS_Status* status);
 CS_Sink CreateCvSinkCallback(std::string_view name,
                              std::function<void(uint64_t time)> processFrame,
                              CS_Status* status);
+CS_Sink CreateCvSinkCallback(std::string_view name,
+                             VideoMode::PixelFormat pixelFormat,
+                             std::function<void(uint64_t time)> processFrame,
+                             CS_Status* status);
```

Comment:
We can do the same thing on the C++ API as we do in Java--only have the pixelFormat-taking functions here and in CvSinkImpl, and only have the default be set in the CvSink class.

Reply:
Would removing the CreateCvSink functions without pixelFormat parameter not be breaking change for code relying upon those functions? Or does all testing code directly call the CvSink constructor?

Reply:
It's technically breaking, but I doubt any teams use the function-based entry points (or if they do, they're experienced enough to track these kinds of changes).

Reply:
Should CvSinkImpl still have a constructor that doesn't use pixelFormat, or only CvSink?

Reply:
Only CvSink.  CvSinkImpl is an implementation only class not visible to users at all.

Reply:
OK that's what is implemented in my last commit